nixos/modules/services/misc/ethminer.nix

```diff
@@ -92,7 +92,9 @@ in
 
       serviceConfig = {
         DynamicUser = true;
+        ExecStartPre = "${pkgs.ethminer}/bin/.ethminer-wrapped --list-devices";
```

Comment:
Test artifact too?

Reply:
this is on purpose, as we have systemd print out the available gpu's, before we start mining.  this is just a sanity check, and to have the list of devices get dumped to systemd log files.